modul/sounds.py

```
@@ -110,9 +120,21 @@ def load_new_sounds(self):
             print(f"Boss spawn sound could not be loaded: {e}")
         try:
             self.boss_death = pygame.mixer.Sound(asset_path("boss_death.wav"))
-        except Exception as e:  # pylint: disable=broad-exception-caught
-            self.boss_death = None
-            print(f"Boss death sound could not be loaded: {e}")
+        except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)